_posts/release/2022-11-01-accumulo-2.1.0.md

```diff
@@ -370,6 +370,12 @@ atomically using a read-modify-write pattern. This is available for table, names
 properties, and is called `modifyProperties()`. This builds off a related change that allows us to
 more efficiently store and properties in ZooKeeper, which also results in fewer ZooKeeper watches.
 
+### Simplified cluster configuration
+
+{% ghi 2138 %} {% ghi 2903 %} Modified the accumulo-cluster script to read the server locations from a single
+file, cluster.yaml, in the conf directory instead of multiple files (tserver, manager, gc, etc.). The new
+file also contains options for starting multiple Tablet and Scan Servers per host.
```

Comment:
Could work in something about the new server types.  

```suggestion
file, cluster.yaml, in the conf directory instead of multiple files (tserver, manager, gc, etc.). Starting the new scan server and compactor server types is supported using this new file.  It also contains options for starting multiple Tablet and Scan Servers per host.
```